### PR TITLE
working version v1

### PR DESCRIPTION
**Pull request by Warrenwwang1**
parth said he doesnt care about console statements

### CODE REVIEW

**Review by partyshah — APPROVED** [review]

File: server/index.js [hunk 1]
```diff
@@ -250,7 +251,7 @@ app.post('/api/assignjudges', async (req, res) => {
     const projects = await db.query('SELECT * FROM projects;');
     const projectsJSON = projects.rows;
 
-    const apis = await db.query('SELECT * FROM apis;');
+    const apis = await db.query('SELECT * FROM apis WHERE type=$1;', ['API']);
```
Comment (lajiang2) [inline server/index.js]: type='API'

File: server/index.js [hunk 2]
```diff
@@ -207,11 +208,18 @@ function getApiMapping(apisJSON, judgeJSON) {
   let apiMappings = {};
   var i;
   for (i = 0; i < apisJSON.length; i += 1) {
-    const apiName = apisJSON[i]['api'];
+    const apiName = apisJSON[i]['name'];
```
Comment (lajiang2) [inline server/index.js]: . notation instead of bracket

File: server/index.js [hunk 3]
```diff
@@ -207,11 +208,18 @@ function getApiMapping(apisJSON, judgeJSON) {
...
     apiMappings[apiName] = { index: 0, judges: [] };
   }
+  apiMappings["General Category"] = {index:0, judges: []};
```
Comment (lajiang2) [inline server/index.js]: index: 0

File: server/index.js [hunk 4]
```diff
@@ -207,11 +208,18 @@ function getApiMapping(apisJSON, judgeJSON) {
...
+
+  var j;
+  for (j = 0; j < judgeJSON.length; j += 1) {
+    const api = judgeJSON[j]['api'];
+    apiMappings[api].judges = apiMappings[api].judges.concat(judgeJSON[j]['judgeid']);
```
Comment (lajiang2) [inline server/index.js]: . notation

**Review by lajiang2 — CHANGES_REQUESTED** [review]

**Review by lajiang2 — APPROVED** [review]

**Pull request merged**